watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -272,41 +272,51 @@ def build(self):
         self.Liq = AqueousPhase()
 
         # list to hold all species (including water)
-        self.component_list = Set()
+        self.component_list = Set(dimen=1)
 
         # components
         self.H2O = Solvent()
 
-        # blank sets
-        self.cation_set = Set()
-        self.anion_set = Set()
-        self.solute_set = Set()
-        self.ion_set = Set()
+        # Other component sets
+        self.solute_set = Set(dimen=1)  # All components except Solvent() ("H2O")
+        self.cation_set = Set(dimen=1)  # Components with charge >0
+        self.anion_set = Set(dimen=1)  # Components with charge <0
+        self.neutral_set = Set(dimen=1)  # Components with charge =0
+        self.ion_set = Set(dimen=1)  # All Ion Components (cations + anions)
 
+        # Group components into different sets
         for j in self.config.solute_list:
+            if j == "H2O":
+                raise ConfigurationError(
+                    "'H2O'is reserved as the default solvent and cannot be a solute."
+                )
+            self.add_component(j, Solute())
+            # This triggers the addition of j into component_list and solute_set
+            # by idaes.core.base.components, although a "del_component" for j will
+            # be called if it goes to cation_ or anion_set (below).
```

Comment:
```suggestion
            # This triggers the addition of j into component_list and solute_set
            # by idaes.core.base.components, although a "del_component" for j will
            # be called if it goes to cation_ or anion_set (below).
            self.add_component(j, Solute())
```

I feel like it's convention to add the comment just before the line called ...

Reply:
Changed the formate and also reworded the note for better clarity.